app/controllers/historic_wdpa_controller.rb

```diff
@@ -0,0 +1,13 @@
+class HistoricWdpaController < ApplicationController
+  def index
+    wdpa_releases = HistoricWdpaRelease.all.order(:year, :month).as_json
+
+    wdpa_releases.each do |release|
+      release[:name] = "#{release['year']} - #{Date::MONTHNAMES[release['month']]}"
```

Comment:
It looks very good, I would only change this to `release['name']` for consistency.

Reply:
Cheers - will change that now :)